### ISSUE TITLE
Serializations to OPML, BML include leading "- "

### ISSUE THREAD

**Issue by RobTrew** [hbsw/tp-issues#281]
This could be by design, but I notice that if we import the .bml serialization into FoldingText for Atom, for example, we have to prune out "- " literals from the start of each item.  

(Equally with OPML in Fargo, Tree, OmniOutliner)

Perhaps having both type="task" and also the leading "- " might be redundant ?

**Comment by jessegrosjean** [comment]
I agree this is a problem, but I don't have time to address at the moment. It needs some new APIs, I'm not exactly sure where to put the logic right now. I will get to this eventually if I make OPML/BML officially supported formats for TaskPaper, but probably not before then.
